src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -167,10 +193,11 @@ protected function single_post_dropdown( $linked_post_id ) {
 	}
 
 	/**
-	 * Render a link to edit the organizer post
+	 * Render a link to edit the linked post
 	 *
-	 * @param int $organizer_id
+	 * @since 3.0
 	 *
+	 * @param int $organizer_id
```

Comment:
If we're updating the docblock in any case ... this should actually be `@param int $linked_post_id` so that it matches the method signature.